packit/constants.py

```diff
@@ -18,9 +18,16 @@
     "packit.yml",
 }
 
+LOCAL_USER_CONFIG_FILE_NAMES = {
+    ".packitLocal.yaml",
+    ".packitLocal.yml",
+    "packitLocal.yaml",
+    "packitLocal.yml",
+}
```

Comment:
I don't really understand why do you hard-code the filenames here, but at the same time you allow the “middle” part to be customized as part of the interface.